src/shared/components/window/WindowStore.ts

```diff
@@ -1,34 +1,3 @@
-import {action, observable} from 'mobx';
-import _ from 'lodash';
-import { bind } from 'bind-decorator';
+import WindowWrapper from "public-lib/components/WindowWrapper";
 
-export interface WindowSize {
-    width: number,
-    height: number
-}
-
-class WindowStore {
-    constructor() {
-        if (typeof window === 'object') {
-            this.windowObj = window;
-            this.setWindowSize();
-            this.windowObj.addEventListener("resize", this.handleWindowResize)
-        }
-    }
-
-    @bind
-    @action
-    private setWindowSize() {
-        this.size = {
-            width: this.windowObj.innerWidth,
-            height: this.windowObj.innerHeight
-        };
-    }
-
-    private handleWindowResize = _.debounce(this.setWindowSize, 200);
-    private windowObj: any;
-
-    @observable size: WindowSize
-}
-
-export default new WindowStore();
\ No newline at end of file
+export default new WindowWrapper();
```

Comment:
separated out the `WindowWrapper` component, but we still use a `singleton` instance for portal frontend, so still keeping the `WindowStore` component here